README.md

```diff
@@ -10,10 +10,14 @@ See [test/index.html](test/index.html) for examples of the SDK in use.
 ### challenge()
 
 When a `Challenge` action is returned from `/checkout`, `/authorize` or
-`/openContract`, call [challenge()](src/reach.lib.js#L68) with the URL included in the API response. 
+`/openContract`, call [challenge()](src/reach.lib.js#L68) with the URL included in the API response.
 ```javascript
 window.rch.challenge(challengeUrl, windowSize, iframeContainer, callback);
-``` 
+```
+You may also pass in a function to call once the challenge screen has loaded.
```

Comment:
This may be getting a bit wordy.  Could we just include `loadedCallback` in the `challenge()` call above and leave the details for the comment block above the function?